.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - "5.3"
```

Comment:
Call me crazy, but unless we're using the super new stuff it brings, we should likely sit on the Node.JS LTS 4.2.x branch. That's what I run right now, as last time I tried 5.x I was greeted with an incredible amount of... friction :confused: